sycl/include/CL/sycl/detail/platform_impl.hpp

```diff
@@ -73,7 +73,7 @@ class platform_impl {
   cl_platform_id get() const {
     if (is_host())
       throw invalid_object_error(
-          "This instance of platform is a host instance");
+          "This instance of platform is a host instance", PI_INVALID_DEVICE);
```

Comment:
```suggestion
          "This instance of platform is a host instance", CL_INVALID_PLATFORM );
```

Reply:
@romanovvlad shouldn't `CL_INVALID_PLATFORM` have a PI version?

Reply:
@alexbatashev I've added this one to the mapping (enum)